test/integration/project_test.go

```diff
@@ -40,7 +40,7 @@ func TestProjectsE2E(t *testing.T) {
 	assert.Equal(t, len(response), 1, "Total projects should be 1")
 	assert.Equal(t, response[0].ID, projectID, "Project ID should match the created project")
 
-	project := showProject(t, projectID)
+	project := response[0]
```

Comment:
show project is another functionality we are testing isn't it? why removing it

Reply:
@tamarleviCm

Reply:
This test is not for testing this command, use it to get the data again (seems unnecessary anyway)
When the flag is on, the group data is not returned because the behavior is different in Access Management
Therefore we used what is returned from the listProjectByID

Reply:
Need to revert this change. need to fix showproject command to have groups with new access management